lib/knapsack_pro/hooks/queue.rb

```diff
@@ -0,0 +1,21 @@
+module KnapsackPro
+  module Hooks
+    class Queue
+      class << self
+        attr_reader :after_subset_queue
+
+        def after_subset_queue(&block)
+          @after_subset_queue ||= block
+        end
```

Comment:
This solution would at least allow us to rename the file that we output the results to. So I think this is actually a fine solution. Because in our spec helper we can do:

```
KnapsackPro::Hooks::Queue.after_subset_queue do |queue_id, subset_queue_id|
  FileUtils.mv('tmp/rspec.xml', "tmp/rspec_#{SecureRandom.hex(10)}.xml")
end